dhis-2/dhis-api/src/main/java/org/hisp/dhis/analysis/FollowupAnalysisParams.java

```diff
@@ -0,0 +1,92 @@
+package org.hisp.dhis.analysis;
+
```

Comment:
I think "analysis" as package name is too generic. Can we instead use the existing "dataanalysis" package?